src/components/InlineEditor.js

```diff
@@ -51,9 +51,12 @@ export default class InlineEditor extends React.Component {
     const changed = initialValue !== value
     if (!editing)
       return (
-        <span className={`editable-field ${value ? (changed ? 'bg-info' : '') : 'placeholder-text'}`} onClick={() => this.setState({ editing: true })}>
+        <span
```

Comment:
I find it odd that this is reformatted. I wonder if your prettier is somehow configured differently...

Reply:
There are some other places where prettier does similar things with my code when I save it. I am not sure exactly how it is supposed be configured and formatted.